packages/headless/src/controllers/url-manager/headless-url-manager.ts

```diff
@@ -40,18 +41,12 @@ export interface UrlManager extends Controller {
    * The state relevant to the `UrlManager` controller.
    * */
   state: UrlManagerState;
+
   /**
    * Updates the search parameters in state with those from the url & launches a search.
    * @param fragment The part of the url that contains search parameters.  E.g., `q=windmill&f[author]=Cervantes`
    */
   synchronize(fragment: string): void;
-  /**
-   * Checks whether the specified fragment matches the value in state.
-   *
-   * @param fragment The part of the url that contains search parameters.  E.g., `q=windmill&f[author]=Cervantes`
-   * @returns `true` if the passed fragment matches the value in state, and `false` otherwise.
-   */
-  hasFragment(fragment: string): boolean;
```

Comment:
I don't think we've released since it was added in a recent PR. We could deprecate it instead.

Reply:
If it was not released officially, just delete.